addons/microsoft_calendar/models/microsoft_sync.py

```diff
@@ -378,7 +378,7 @@ def _sync_microsoft2odoo(self, microsoft_events: MicrosoftEvent):
                 ms_event_updated_time = parse(mevent.lastModifiedDateTime)
 
                 if ms_event_updated_time >= odoo_event_updated_time:
-                    vals = dict(odoo_event._microsoft_to_odoo_values(mevent), need_sync_m=False)
+                    vals = dict(odoo_event.with_context(update_edge_case=True)._microsoft_to_odoo_values(mevent), need_sync_m=False)
```

Comment:
update_edge_case seems a bit weird. Because the edge case is the combination of this context value as truthy with the update of start/stop. This context value represents only that the ms_event is more up to date than the odoo event.

Reply:
Yes, or 'ms_event_is_most_recent'